packages/tests/schema-misc/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "joist-tests-schema-misc",
+  "version": "1.0.0",
+  "license": "MIT",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stephenh/joist-ts.git",
+    "directory": "packages/tests/schema-misc"
+  },
+  "scripts": {
+    "migrate": "DATABASE_URL=postgres://joist:local@localhost:5435/schema_misc node-pg-migrate up --decamelize=false --migrations-dir=./migrations",
```

Comment:
I'm running `node-pg-migrate` directly to turn off `decamelize` so that we can test camel-cased names in the db.